src/site/content/en/metrics/inp/index.md

```diff
@@ -14,12 +14,14 @@ tags:
 ---
 
 {% Aside %}
-Interaction to Next Paint (INP) is an experimental metric that assesses [responsiveness](/user-centric-performance-metrics/#types-of-metrics). INP observes the latency of all interactions a user has made with the page, and reports a single value which all (or nearly all) interactions were below. A low INP means the page was consistently able to respond quickly to all&mdash;or the vast majority&mdash;of user interactions.
+Interaction to Next Paint (INP) is an experimental metric that assesses [responsiveness](/user-centric-performance-metrics/#types-of-metrics). When an interaction causes a page to become unresponsive, that is a poor user experience. INP observes the latency of all interactions a user has made with the page, and reports a single value which all (or nearly all) interactions were below. A low INP means the page was consistently able to respond quickly to all&mdash;or the vast majority&mdash;of user interactions.
 {% endAside %}
 
 Chrome usage data shows that 90% of a user's time on a page is spent _after_ it loads, Thus, careful measurement of responsiveness _throughout_ the page lifecycle is important. This is what the INP metric assesses.
 
-Good responsiveness means that a page responds quickly to interactions made with it. When a page responds to interactions, the changes in the user interface that result are _visual feedback_ provided in the next frame the browser presents. Visual feedback tells us if, for example, an item we've asked to add to an online shopping cart is indeed being added, if a login form's contents are being authenticated by the server, whether a mobile menu has opened, and so on.
+Good responsiveness means that a page responds quickly to interactions made with it. When a page responds to an interaction, the result is _visual feedback_, which is presented by the browser in the next frame the browser presents. Visual feedback tells you if, for example, an item you add to an online shopping cart is indeed being added, whether a mobile navigation menu has opened, if a login form's contents are being authenticated by the server, and so forth.
+
+Some interactions will naturally take longer than others, but for especially complex interactions, it's important to quickly present some initial visual feedback as a cue to the user that something is _happening_.  The time until the next paint is the earliest opportunity to do this. Therefore, the intent of INP is not to measure the total time of the interaction, but the time in which the next paint is being blocked. By delaying visual feedback, you may be giving users the impression that the page is not responding to their actions.
```

Comment:
Super nit-picky, but: "the intent of INP is not to measure the total time of the interaction"

I think this is sorta circular in that it depends on your definition of Interaction and total time.  We certainly in other contexts do say things like: "compared to FID the goal of INP is to measure total duration..." which can easily be seen as conflicting here.

Perhaps to make it very clear, you could expand to say like "the intent of INP is not to measure all of the eventual effects of the interaction (i.e. network fetches) ..."

But I leave it to you, its certainly fine as is as well.

Reply:
I think it's "compared to FID the goal of INP is to measure total duration... _until that first paint_".

I think that's very different than saying "we're going to measure the total duration of this interaction" - which was my (incorrect) understanding until you explained it to me :-)

Reply:
Absolutely yes.  It also gets even more confusing when the first paint has no visual update..., (or it does, but the user doesn't perceive that first paint to have a visual update).

Net/net, INP ensures that you could have provided useful visual feedback, not that you actually did.